src/App.jsx

```diff
@@ -1,5 +1,41 @@
 import 'bulma/css/bulma.css';
 import './App.scss';
+import cn from 'classnames';
+import { useState } from 'react';
+
+const SORT_BY_NAME = 'alphabet';
+const SORT_BY_LENGTH = 'length';
+const REVERSE_ORDER = 'reversed';
+let isNameSelected = false;
+let isLengthSelected = false;
+let isReverseSelected = false;
+
+function getPreparedGoods(goods, sortField, reverseSelection) {
+  const preparedGoods = [...goods];
+
+  preparedGoods.sort((good1, good2) => {
```

Comment:
let's use 'a' and 'b' instead of good1 and good2, using numbers in variable names is not the best practice